### ISSUE TITLE
Initialize for new rows

### ISSUE THREAD

**Issue by zhudock** [Pretius/apex-enhanced-lov-item-tabular-form#1]
Is is possible to initialize this plugin for a new row as well, when added with `apex.widget.tabular.addRow();`

**Comment by TodGoulds** [comment]
i have the same question: Is is possible to initialize this plugin for a new row as well, when added with apex.widget.tabular.addRow();

**Comment by TodGoulds** [comment]
So Bartosz Ostrowski showed me how to do this.

1.Set static ID of region: <exampleLov>

2.make new dynamic action
Event: Custom
Custom Event: mutationObserver
Selection Type: Region
Region: <your region name>
True Action: Pretius APEX Enhanced LOV item
Plugin instances: <item name>
Column names: <fx...>

3. enter "Function and Global Variable Declaration" on page

var 
  MutationObserver = window.MutationObserver || window.WebKitMutationObserver || window.MozMutationObserver,
  reportId = '#exampleLov',
  report = $(reportId),
  reportObservedElem = document.querySelector( reportId+' .t-Report-report tbody' ); 
 
var observer = new MutationObserver(function(mutations) {
  mutations.forEach(function(mutation) {
    if (mutation.type === 'childList') {
      var fakeDiv = report.find('.paeli_tf-container');
      fakeDiv.each( function(){
        var 
          self = $(this),
          realItem = self.find('.fakePluginReturn');
        realItem.removeClass('fakePluginReturn').detach();
        realItem.appendTo( self.parent() );
        realItem.removeAttr('style');
        self.remove();
      } );
      apex.event.trigger(report, 'mutationObserver');
    }
  });
});
observer.observe(reportObservedElem, {
  childList: true
});
